README.md

```
@@ -33,21 +35,103 @@ gem 'kessel-sdk'
 gem 'openid_connect', '~> 2.0'  # Optional - only for authentication
 ```
 
+The `openid_connect` gem is intentionally not a hard runtime dependency. Consumers who do not need OAuth are not forced to install it or its transitive dependencies.
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)